modules/installation-and-upgrade/pages/db-migration-xy.adoc

```diff
@@ -10,7 +10,7 @@ If you want to upgrade to the latest {productname} version, you must be using P
 * If you are running {opensuse} Leap 15.4, use PostgreSQL 14.
 
 * If you are running {sles} 15 SP5, use PostgreSQL 15.
-* If you are running {opensuse} Leap 15.5, use PostgreSQL 15.
+* If you are running {opensuse} Leap 15.5, use PostgreSQL 14.
```

Comment:
But besides, it says above:

> If you are already using PostgreSQL 14, you do not need to perform this migration.

But this whole section looks strange to me?


> This section covers upgrading the PostgreSQL database to the latest version.
> If you are already using PostgreSQL 14, you do not need to perform this migration.

Is this about the server or clients?

Because if it's about the server, as it looks to be:

> If you want to upgrade to the latest {productname} version, you must be using PostgreSQL version 13, 14, or 15 depending on the underlying operating system:

Why 15? You should be using 14 already.

> * If you are running {sles} 15 SP4, use PostgreSQL 14.

Why do we mention SLE?

> * If you are running {opensuse} Leap 15.4, use PostgreSQL 14.

Don't understand every well

> * If you are running {sles} 15 SP5, use PostgreSQL 15.
> * If you are running {opensuse} Leap 15.5, use PostgreSQL 15.
> * If you are running {opensuse} Leap 15.5, use PostgreSQL 14.

Same about this. In the end all users should already be in Leap 15.4 and using PostgreSQL 14.

Reply:
ATM, seems like nothing to do. There are no versions to migrate to or from as we are staying on postgresql 14. I would remove the version 13 comment. Maybe even comment this file out until we require it for version 15?